clients/drcachesim/common/trace_entry.h

```diff
@@ -191,6 +191,24 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_CPU_ID,
 
+    /**
+     * The marker value contains identifier for the return address of
+     * malloc-on-heap-like function
```

Comment:
I think the return address can be treated as the calling site, right? So in the case of malloc, it would be the allocation site of an object, which is probably very useful for later analysis.

Reply:
As mentioned, some applications have layers here: utilities like Chrome's heap allocation all directed through a single place so they can pick which allocator to use or take other common actions.  There, a single retaddr is not useful.  We could start with this but please put in a XXX comment about potentially changing this to a callstack.

Reply:
Sure, will put the comment as so.

Reply:
Done